package.json

```diff
@@ -136,7 +136,7 @@
     "@types/lingui__react": "^2.8.3",
     "@types/ms.macro": "^2.0.0",
     "@types/multicodec": "^1.0.0",
-    "@types/node": "^13.13.5",
+    "@types/node": "^16.7.13",
```

Comment:
you might want to update `.nvmrc` to be 16 too (lts is 16 rn, but will be 18 soon)